proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/SamlResponseExceptionMapperResourceRuleTest.java

```diff
@@ -44,10 +44,9 @@ public void before() {
                 new SamlFormView("postUrl",
                                  "samlMessageType",
                                  "encodedSamlMessage",
-                                 "submitText"));
+                                 "relayState"));
```

Comment:
This is interesting - it looks like you've fixed a bug. (My understanding was that the `SamlFormView` constructor would have dropped from 5 params to 4 params (removing the `submitText`), but this went from 4 to 4...

Reply:
Yep - there were some changes made a while ago to the SAML form view and not everything was cleaned up properly then